x/group/conditions.go

```diff
@@ -0,0 +1,126 @@
+package group
```

Comment:
This needs to be replaced with ADR 028 eventually

Reply:
Is there an issue to track this @blushi ? I think we have a clearer sense of what we'll be doing now.

Reply:
Oh nevermind, there is...